UPGRADING.md

```diff
@@ -17,7 +17,10 @@ This guide provides instructions for upgrading to specific versions of Tendermin
 
 ### Config Changes
 
-* `fast_sync = "v1"` and `fast_sync = "v2"` are no longer supported. Please use `v0` instead.
+* The configuration variable `fast-sync` and flag `--fast-sync` have been renamed to `enable-block-sync` and `--enable-block-sync`
+  and the field configuration field `[fastsync]` has been renamed to `[blocksync]`.
+
+* `blocksync.version = "v1"` and `blocksync.version = "v2"` (previously `fastsync`) are no longer supported. Please use `v0` instead.
```

Comment:
Not specific to your change, but is it not a little weird that we are advocating what looks like a version-stamp _downgrade_ here? I assume this isn't a typo, but should we explain further?

Reply:
This is a great question, I don't have context on why this change was made, I wonder if perhaps @cmwaters may?

Reply:
cc: @marbar3778

Reply:
the pkg was rewritten 2 times, they were named v1 and v2. They were written by the informal team, but soon after completion they started working on tm-rs. We knew v1 would be removed and the goal was to migrate to v2, but issues arose and the team thought the overhead of understanding v2 was too great. A large reason for this is that the new versions didnt add much benefit over v0.

Reply:
I've added some language here that may capture the intention behind removing, please take a look if this seems correct.

Reply:
looks good!! thanks for adding it.